arch/arm/boot/dts/overlays/arducam-pivariety-overlay.dts

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Definitions for Arducam Pivariety camera module on VC I2C bus
+/dts-v1/;
+/plugin/;
+
+#include <dt-bindings/gpio/gpio.h>
```

Comment:
This include is spurious - no GPIO bindings are used.

Reply:
It has been fixed.